Sources/IBLinterKit/Config.swift

```diff
@@ -39,17 +41,22 @@ public struct Config: Codable {
         included = []
         customModuleRule = []
         useBaseClassRule = []
+        viewAsDeviceRule = nil
         reporter = "xcode"
         disableWhileBuildingForIB = true
     }
 
-    init(disabledRules: [String], enabledRules: [String], excluded: [String], included: [String], customModuleRule: [CustomModuleConfig], baseClassRule: [UseBaseClassConfig], reporter: String, disableWhileBuildingForIB: Bool = true) {
+    init(disabledRules: [String] = [], enabledRules: [String] = [],
+        excluded: [String] = [], included: [String] = [],
+        customModuleRule: [CustomModuleConfig] = [], baseClassRule: [UseBaseClassConfig] = [], viewAsDeviceRule: ViewAsDeviceConfig? = nil,
+        reporter: String = "", disableWhileBuildingForIB: Bool = true) {
```

Comment:
```suggestion
        reporter: String = "xcode", disableWhileBuildingForIB: Bool = true) {
```

The default value of `reporter` shouldn't be empty string because it's invalid reporter name. I prefer to use "xcode" instead.